rundeckapp/grails-spa/packages/ui/src/pages/webhooks/i18n.js

```diff
@@ -20,6 +20,11 @@ const translationStrings = {
       webhookUserHelp:'The authorization username assumed when running this webhook. All ACL policies matching this username will apply.',
       webhookRolesLabel:'Roles',
       webhookRolesHelp:'The authorization roles assumed when running this webhook (comma separated). All ACL policies matching these roles will apply.',
+      webhookAuthLabel: 'HTTP Authorization String',
```

Comment:
suggest labelling it "HTTP Authentication Header" instead of "String"
```suggestion
      webhookAuthLabel: 'HTTP Authentication Header',
```